src/lib/core/CHIPVendorIdentifiers.hpp

```diff
@@ -26,17 +26,19 @@
 
 #pragma once
 
+#include <cstdint>
+
 namespace chip {
 
 //
 // CHIP Vendor Identifiers (16 bits max)
 //
 
-enum ChipVendorId
+enum VendorId : uint16_t
 {
-    kChipVendor_Common       = 0x0000,
-    kChipVendor_NestLabs     = 0x235A,
-    kChipVendor_NotSpecified = 0xFFFF
+    Common       = 0x0000u,
+    NestLabs     = 0x235Au,
```

Comment:
For separte PR: do we use NestLabs anywyere?

Reply:
We do not.  I was figuring I should check whether this vendor id is in fact assigned to Nest Labs before removing, though....

Reply:
We do use this VendorID in efr examples. It was used for a demo with Google Hub Nest max and it stayed like this.
#define CHIP_DEVICE_CONFIG_DEVICE_VENDOR_ID 0x235A

Probably not needed anymore.